unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/DCLTexture/AssetPromise_DCLTexture.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using UnityEngine;
+
+namespace DCL
+{
+    public class AssetPromise_DCLTexture : AssetPromise<Asset_DCLTexture>
```

Comment:
I have a big concern with this rename:

1. Is making this PR diff hard to follow
2. Why `DCLTexture`? All the other AssetPromise don't have the `DCL` prefix on it and they don't need it.
3. `DCLTexture` on the component name was needed to avoid naming conflict with `UnityEngine.Texture`. This is not the case.

Reply:
At the beginning it was going to be an asset promise keeper of a DCLTexture and in the end it is doing that purpose somehow. However, I will call it `TextureResource `if that is ok with you!